fcl/ValenciaMECq0q3InterpWeighting.PH.fcl

```diff
@@ -0,0 +1,119 @@
+generated_systematic_provider_configuration: {
+   ValenciaMECq0q3InterpWeighting_SBNNuSyst: {
+      ValenciaMECResponse: {
+         centralParamValue: 0
+         isSplineable: true
+         paramVariations: [
+            1
+         ]
+         prettyName: "ValenciaMECResponse"
+         systParamId: 0
+      }
+      instance_name: "SBNNuSyst"
+      parameter_headers: [
+         "ValenciaMECResponse"
+      ]
+      tool_options: {
+         ValenciaMECResponse_input_manifest: {
+            EnergyGrid: [
+               4e-1,
+               5e-1,
+               6e-1,
+               7e-1,
+               8e-1,
+               9e-1,
+               1,
+               1.1,
+               1.2,
+               1.3,
+               1.4,
+               1.5,
+               1.6,
+               1.7,
+               1.8,
+               1.9,
+               2,
+               2.1,
+               2.2,
+               2.3,
+               2.4,
+               2.5
+            ]
+            WeightLimits: [
+               0.1,
+               1000.0
+            ]
+
+            # NEW: apply only in this q0 window
+            Q0ApplyMin: 0.0     # GeV
+            Q0ApplyMax: 3.0     # GeV
+
+            Q0GuardMin: 0.0            
+            Q0GuardMax: 0.0 
+
+            EnuMin:       0.4
+            EnuMax:       2.5
+            EnuSnapTol:   0.005
```

Comment:
These part are not included in the ToolConfig (`ValenciaMECq0q3InterpWeighting.ToolConfig.fcl`), and if you want users to include these parameters, can you also update it? Btw, it is always best to first edit "ToolConfig", then run `GenerateSystProviderConfigNuSyst` on it to create "ParameterHeader (PH)" file.